.github/workflows-source/ci-main.yml

```diff
@@ -64,11 +68,21 @@ jobs:
   bazel-test-all:
     name: Bazel Test All
     <<: *dind-large-setup
-    env:
-      TITLE: ${{ github.event.pull_request.title }}
     steps:
       - <<: *checkout
       - <<: *before-script
+      - name: check trigger type
```

Comment:
It's good that we're consistent. If all other names start with capita, it makes sense to have "Check trigger type" here.

Reply:
And maybe better to name it "Setting BAZEL_EXTRA_ARGS_RULES".